sources/platform/tutorial/workflow/sending-status-to-scm.md

```diff
@@ -27,7 +27,7 @@ jobs:
 
 ```
 
-The corresponding  `gitRepo` resource in `shippable.resources.yml` needs to include the tag `buildOnPullRequest: true` as shown below:
+The corresponding  `gitRepo` resource in `shippable.resources.yml` needs to include the tag `buildOnPullRequest: true` and `buildOnPullRequestClose: true`as shown below:
```

Comment:
i dont think it should be added here, this implies that it is required in order to get build status on your PR, but that's not the case.